pages/common/docker.md

```diff
@@ -18,3 +18,11 @@
 - Stop a container
 
 `docker stop {{container}}`
+
+- List all images
```

Comment:
Not very useful since it's right there in the command's help. A useful tldr would be `docker images --filter="dangling=true"` for all untagged images, or `docker images -qa` for all images ids (ideal for a cleanup).